Core.ApplicationServices/UserService.cs

```diff
@@ -280,5 +289,51 @@ public void Dispose()
                     )
                 );
         }
+
+        public Maybe<OperationError> DeleteUserFromKitos(Guid userUuid)
+        {
+            using var transaction = _transactionManager.Begin();
+
+            var user = _userRepository.AsQueryable().ByUuid(userUuid);
+            if (user == null)
+                return Maybe<OperationError>.Some(new OperationError(OperationFailure.NotFound));
```

Comment:
no need to add Maybe.Some.. returning a new operation error will do the wrapping itself